### PR TITLE
Shunting

### PR DESCRIPTION
**Pull request by hm-lnkd**
I have used the same token class mentioned prior and most of the functionality compares the object to what class type it is or is not as well as what precedent value it contains.

### CODE REVIEW

File: calculator/src/ShuntingYard.kt [hunk 1]
```diff
@@ -0,0 +1,52 @@
+
+
+fun shunt (inputList: List<Token>) : List<Token>
+{
+    var outputQueue= mutableListOf<Token>()
+    var operatorStack= mutableListOf<Token>()
```
Comment (Nicholas-Baron) [inline calculator/src/ShuntingYard.kt]: I do not see any reassignments. Use `val` to signify this.

File: calculator/src/ShuntingYard.kt [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    for(x in inputList)
+    {
+        if(x is Floating || x is Integer)
+            outputQueue.enqueue(x)
+        else if(x is Function)
+            operatorStack.push(x)
+        else if(x is BinaryOp) {
```
Comment (Nicholas-Baron) [inline calculator/src/ShuntingYard.kt]: Inconsistent formatting?

PS Intellij has Ctrl-Atl-L to auto format

**Review by Nicholas-Baron — CHANGES_REQUESTED** [review]
Looks decent

File: calculator/src/ShuntingYard.kt (hunk 1 above)
Comment (hm-lnkd) [reply calculator/src/ShuntingYard.kt]: resolved and pushed new shunting file

File: calculator/src/ShuntingYard.kt (hunk 2 above)
Comment (hm-lnkd) [reply calculator/src/ShuntingYard.kt]: formatted!

**Review by Nicholas-Baron — APPROVED** [review]

**Pull request merged**